src/plugins.js

```diff
@@ -1,8 +1,10 @@
+import config from './util/config';
```

Comment:
config returns to the plugins script - mainly to make sure the tracking plugin is not dependent on global config (it only needs 1 `isProd` property)